recipes/r-saige/conda_build_config.yaml

```diff
@@ -1,4 +1,5 @@
 r_base:
-  - 3.6.1
+  - 3.6.3
```

Comment:
Do we absolutely need to support older R versions? There will be a new one in a couple months.

Reply:
Is it causing too much issues? If there is a reasonable point I am ok to drop. I was keeping it just because I still get researchers asking for some older version due to some other packages that are not being updated, and it's just hell to install tools like this one manually.. And it is also not in my powers to track down all those other packages.

Reply:
Fair enough, it's just easier in a few months to not have manually track down cases like this when R 4.1 comes out and I rebuild all of the R packages.

Reply:
I wish there were like LTS versions for R, e.g. at least two major versions being built by main config - not necessarily having configs for this in repo, then it could be easier to switch all.

Reply:
The main rate limiter for this is Bioconductor. If their packages were meant to support more than a single minor version at a time then this would be much easier.